index.bs

```diff
@@ -792,6 +814,80 @@ option. If <code><a for="underlying source">type</a></code> is set to <code>unde
   </code></pre>
 </div>
 
+<!-- Bikeshed doesn't let us mark this up correctly: https://github.com/tabatkins/bikeshed/issues/1344 -->
+<h5 id="rs-asynciterator" iterator for="ReadableStream">[@@asyncIterator]({ <var>preventCancel</var> } = {})</h5>
```

Comment:
Using `iterator` instead of `method` here stops Bikeshed from checking if the content is a valid method name, while still doing the linking. (`attribute` also works.)

A bit of a hack, but good enough for now until Bikeshed lets us do this properly? 😅

Reply:
Seems fine to me. Also, it looks like you did it on purpose, which is a bonus.

Reply:
We shouldn't be adding attributes that do nothing. I'll work on a better version.

Reply:
Oh, `iterator` is a valid Bikeshed definition type. Not bad, I'll take it!